views/homework/extended.hbs

```diff
@@ -84,4 +84,9 @@
     </div>
     {{/content}}
     {{/embed}}
-</section>
\ No newline at end of file
+    {{#embed "lib/components/modal-information" class="danger-modal" title="Achtung"}}
+	    {{#content "fields"}}
+		    {{> "files/forms/form-virus-danger"}}
+	    {{/content}}
+    {{/embed}}

```

Comment:
--> please move it after the section tag

Reply:
@dominikbruechner the other modal is also implemented above the section. So I would go for consistency and keep it there.